src/platform/packages/shared/deeplinks/security/deep_links.ts

```diff
@@ -10,6 +10,8 @@
 export enum SecurityPageName {
   administration = 'administration',
   alerts = 'alerts',
+  attacks = 'attacks',
+  alertDetections = 'alert_detections',
```

Comment:
Maybe a dumb question, but do you think we could use `detections` directly? I understand that we don't want to use it in production as we've been using it before and we could have issues with rerouting and such, but adding some code that we know from the get go will have to be refactored soon isn't ideal either.

If we want to stick with `alert_detections` for now, then at least let's add a TODO here with the link you created (https://github.com/elastic/kibana/issues/242434) to make it easier to clean things up when we're ready.